lua/telescope/actions/init.lua

```diff
@@ -358,6 +358,31 @@ actions.git_checkout = function(prompt_bufnr)
   end
 end
 
+--- Switch to git branch
+--- If the branch already exists in local, switch to that.
+--- If the branch is only in remote, create new branch tracking remote and switch to new one.
+---@param prompt_bufnr number: The prompt bufnr
```

Comment:
Make sure github actions for your fork are enabled so docgen is running. We are also running into a maximum amount of docs we can generate for that file (some bug in treesitter) so we are currently documenting new functions like this: 
```lua
-- TODO: add this function header back once the treesitter max-query bug is resolved
--
--
```
With two rather than 3 `-`. See 379